CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ else()
   option(DISABLE_DEPRECATION_WARNINGS "Disable depreaction warnings " ON)
   option(NVTX "Enable nvtx markers" OFF)
   set(BACKEND_FOLDER "/opt/tritonserver/backends" CACHE STRING "Triton backend folder path")
-  set(RAPIDS_DEPENDENCIES_VERSION "22.06" CACHE STRING "RAPIDS projects dependencies version")
+  set(RAPIDS_DEPENDENCIES_VERSION "22.08" CACHE STRING "RAPIDS projects dependencies version")
```

Comment:
Since we are using a 22.10 commit, shouldn't this be a hash in rapids-cmake 22.10 as well?

Reply:
I could go either way on this. My take on it was that I'd rather use the minimum amount of nightly code, so we set cuML to the appropriate commit and then keep everything else as it would have been if we weren't trying to get that specific code from cuML.

Reply:
As long as it works, rapids-cmake 22.08 is frozen so it should be fine